baseline/OSPS-VM.yaml

```
@@ -297,3 +164,144 @@ controls:
           To the degree possible, this information should include affected
           version(s), how a consumer can determine if they are vulnerable, and
           instructions for mitigation or remediation.
+      - id: OSPS-VM-04.02
+        text: |
+          The project's released software assets MUST include VEX data to
+          provide information about the exploitability of vulnerabilities.
```

Two notes here:
1. This should be a MUST only when dealing with false positives. 

2. You can include a snapshot of the applicable data in the assets, but the VEX feed keeps moving after the release, so we prefer to have a continuously updating feed.

```suggestion
          Any vulnerabilities in the software components not affecting the
          project MUST be accounted for in a VEX document, augmenting
          the vulnerability report with non-exploitability details.
```

---

thanks for this! Working to refine it... Where should the VEX document live? in any part of the project documentation, or specifically alongside the corresponding release assets?